src/main/pages/setup-docker/docker-single-user.adoc

```diff
@@ -157,123 +161,130 @@ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock
+. Initialize the new version.
+. Start Che.
 
-1.  Performs a version compatibility check
-2.  Downloads new Docker images that are needed to run the new version of Che
-3.  Stops Che if it is currently running
-4.  Triggers a maintenance window
-5.  Backs up your installation
-6.  Initializes the new version
-7.  Starts Che
-8.  Important! If `CHE_PREDEFINED_STACKS_RELOAD__ON__START` is set to false, stacks packaged into new binaries will not be saved into a database.
+[IMPORTANT]
+====
+If the `CHE_PREDEFINED_STACKS_RELOAD__ON__START` variable is set to false, stacks packaged into the new binaries will not be saved in the database.
+====
 
 [id="backup"]
-== Backup
+== Taking a backup
 
-You can run `che backup` to create a copy of the relevant configuration information, user data, projects, and workspaces. We do not save workspace snapshots as part of a routine backup exercise. You can run `che restore` to recover Che from a particular backup snapshot. The backup is saved as a TAR file that you can keep in your records. You can then use `che restore` to recover your user data and configuration.
+* To create a copy of the relevant configuration information, user data, projects, and workspaces, run the `che backup` command. As part of a routine backup exercise, Che does not save workspace snapshots. 
+* To recover Che from a particular backup snapshot, run the `che restore` command. The backup is saved as a TAR file that you can keep in your records. You can then use the `che restore` command to recover your user data and configuration.
 
 [id="configuration"]
 == Configuration
 
-Che CLI allows a wide range of config changes to setup port, hostname, oAuth, Docker, git, and solve networking issues. See: link:docker-config.html[Che configuration on Docker].
+Che CLI allows a wide range of configuration changes to setup the port, hostname, oAuth, Docker, git, and to solve networking issues. For details, see link:docker-config.html[Che configuration on Docker].
 
 [id="run-without-cli"]
-== Run Without CLI
+== Running Che without the CLI
+
+You can run the Che server directly by launching a Docker image. This approach bypasses the CLI that has additional utilities to simplify administration and operation. The `eclipse/che-server` Docker image is appropriate for running Che within clusters, orchestrators, or by third-party tools with automation.
 
-You can run the Che server directly by launching a Docker image. This approach bypasses the CLI, which has additional utilities to simplify administration and operation. The `eclipse/che-server` Docker image is appropriate for running Che within clusters, orchestrators, or by third party tools with automation.
+To run Che directly by launching a Docker image, take the following steps:
 
+. Run the latest released version of Che.
+. Replace <LOCAL_PATH> with any host folder. Che places the backup files in this folder - configurable properties, workspaces, lib, storage.
```

Comment:
* mark up &lt;LOCAL_PATH&gt;
* configurable properties, workspaces, lib, storage -- what are these? files? directories? backed up data?

Reply:
Changed to "Replace `<LOCAL_PATH>` with any host folder. Che places the backup files (configurable properties, workspaces, lib, storage) in this folder." Better?

Also, @rkratky , I don't know the answers to a few of your comments. I don't want to guess and mess it up. Who can I ask for explanation, please?

Reply:
> Changed to "Replace <LOCAL_PATH> with any host folder. Che places the backup files (configurable properties, workspaces, lib, storage) in this folder." Better?

We need to unify folder vs directory. I think 'directory' is better.

> Also, @rkratky , I don't know the answers to a few of your comments. I don't want to guess and mess it up. Who can I ask for explanation, please?

We're working on it. In the meantime, @eivantsov is our man. @eivantsov, please, feel free to delegate SME queries.

Reply:
> > Changed to "Replace <LOCAL_PATH> with any host folder. Che places the backup files (configurable properties, workspaces, lib, storage) in this folder." Better?
> 
> We need to unify folder vs directory. I think 'directory' is better.

Okay. I'll change all instances of "folder" to "directory".
> 
> > Also, @rkratky , I don't know the answers to a few of your comments. I don't want to guess and mess it up. Who can I ask for explanation, please?
> 
> We're working on it. In the meantime, @eivantsov is our man. @eivantsov, please, feel free to delegate SME queries.
Sure. Thanks!